nanoFramework.System.Threading/packages.config

```diff
@@ -1,8 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Build.Tasks.Git" version="1.0.0" targetFramework="netnanoframework10" developmentDependency="true" />
```

Comment:
These NuGet packages are meant to be on the packages file and are used in the build.
Please don't remove them.
Microsoft.Build.Tasks.Git
Microsoft.SourceLink.Common
Microsoft.SourceLink.GitHub
Nerdbank.GitVersioning